### PR TITLE
new file:   packages/molenc/molenc.15.4.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/molenc/molenc.15.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,75 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/molenc"
+bug-reports: "https://github.com/UnixJunkie/molenc/issues"
+dev-repo: "git+https://github.com/UnixJunkie/molenc.git"
+license: "BSD-3"
```
Comment (mseri) [inline packages/molenc/molenc.15.4.0/opam:7]:
```suggestion
license: "BSD-3-Clause"
```

File: packages/molenc/molenc.15.4.0/opam [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  "bst"
+  "conf-graphviz"
+  "conf-python-3"
+  "conf-rdkit"
+  "cpm"
+  "dokeysto_camltc"
+  "dolog" {>= "5.0.0"}
+  "dune" {>= "1.11"}
+  "minicli" {>= "5.0.0"}
+  "ocaml"
```
Comment (mseri) [inline packages/molenc/molenc.15.4.0/opam:34]:
```suggestion
  "ocaml" {>= "4.07"}
```

Module `Float` was introduced in 4.07

File: packages/molenc/molenc.15.4.0/opam [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
```
Comment (mseri) [inline packages/molenc/molenc.15.4.0/opam]:
```suggestion
  "ocaml" {>= "4.07"}
```
Module `Float` was introduced in 4.07. Do you want to support older compilers or this is fine?

**Comment by mseri** [comment]
Looks good now. If you are happy with the changes, please let me know and I'll merge (also please upstream them)

**Comment by UnixJunkie** [comment]
Of course, I am ok with your corrections.
Thanks a lot!
I will upstream them.

**Pull request merged**